BookingsAPI/Bookings.IntegrationTests/Steps/SuitabilityAnswersSteps.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Bookings.Api.Contract.Responses;
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+using Testing.Common.Builders.Api;
+
+namespace Bookings.IntegrationTests.Steps
+{
+    [Binding]
+    public sealed class SuitabilityAnswersSteps : StepsBase
+    {
+        private readonly SuitabilityAnswerEndpoints _endpoints = new ApiUriFactory().SuitabilityAnswerEndpoints;
+
+        public SuitabilityAnswersSteps(Contexts.TestContext apiTestContext) : base(apiTestContext)
+        {
+        }
+
+        [Given(@"I have a get suitable answers for participants")]
```

Comment:
this given sentence is a bit hard to understand exactly what it is, can it be turned around maybe? something like "I have a request to list suitability answers"

Reply:
No, it means the participant has answers, I will change it to "I have a participant with suitability answers"?

Reply:
Ah, yes, that'll be a lot better!